nifi-nar-bundles/nifi-marklogic-bundle/LICENSE.txt

```diff
@@ -0,0 +1,72 @@
+Apache License 

```

Comment:
This license file should be located in the NAR under meta-inf just like the following example.  Further, it should contain the same text/same format of text for the ALv2 license.

https://github.com/apache/nifi/blob/master/nifi-nar-bundles/nifi-azure-bundle/nifi-azure-nar/src/main/resources/META-INF/LICENSE

Reply:
Also unless you're altering the license by appending items to it as would be necessary was there any copied source from somewhere and that source was from the ASF category A you dont need to include it as it will be pulled in automatically.